src/gsy_e/models/area/markets.py

```diff
@@ -117,6 +117,7 @@ def _activate_future_markets(self, area: "Area") -> None:
                               grid_fee_const=area.grid_fee_constant),
             name=area.name)
         self.future_markets = market
+        self.future_markets.update_clock(area.now)
         area.dispatcher.create_market_agents_for_future_markets(market)
 
     def _activate_forward_markets(self, area: "Area") -> None:
```

Comment:
Should we add similar change for _activate_forward_markets? I was not sure.

Reply:
I think so, since the forward markets inherit from the future market, therefore in theory if one of these markets posts an offer on the first tick it will not populate the creation time correctly.